climada/engine/unsequa/calc_base.py

```diff
@@ -500,10 +536,44 @@ def _calc_sens_df(method, problem_sa, sensitivity_kwargs, param_labels, X, unc_d
         else:
             sens_indices = method.analyze(problem_sa, Y,
                                                     **sensitivity_kwargs)
+        #refactor incoherent SALib output
+        nparams = len(param_labels)
+        if method.__name__ == 'SALib.analyze.ff':
```

Comment:
Because it is not available from within _calc_sens_df, I think? But we can pass it as an argument of _calc_sens_df if you think it is worth it.

Reply:
But it is already... the argument `method` is a string.

Reply:
Ok then I changed it to only check if the corresponding last characters of the method name string correspond to the target method name instead of the entire `Salib.analyse.ff`.